internal/receiver/scriptedinputsreceiver/config.go

```diff
@@ -96,6 +97,10 @@ func (c *Config) Validate() error {
 
 // Build will build a stdoutOperator.
 func (c *Config) Build(logger *zap.SugaredLogger) (operator.Operator, error) {
+	if isContainer() {
+		return nil, fmt.Errorf("cannot run scripted inputs receiver in Docker container")
```

Comment:
Do you mind updating the readme for this limitation as well?

```suggestion
		return nil, fmt.Errorf("scriped inputs receiver must be run directly on host and is not supported in container.")
```